__tests__/jubilation-line/jubilation-line-test.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+import JubilationLine from '../../src/jubilation-line';
+import JubilationProvider from '../../src/jubilation-provider';
+
+describe('JubilationLine', () => {
+  it.skip('should expect x1, y1, x2, y2 props', () => {
+    // TODO spy on console.log to check for React warning message
```

Comment:
See: http://stackoverflow.com/questions/26124914/how-to-test-react-proptypes-through-jest